azure-pipelines.yml

```diff
@@ -252,8 +257,8 @@ stages:
           TOPOLOGY: dpu
           MIN_WORKER: $(T0_SONIC_INSTANCE_NUM)
           MAX_WORKER: $(T0_SONIC_INSTANCE_NUM)
-          KVM_IMAGE_BRANCH: "master"
-          MGMT_BRANCH: "master"
+          KVM_IMAGE_BRANCH: $(BUILD_BRANCH)
```

Comment:
It was a wrong config, buildimage PR test doesn't need KVM_IMAGE_BRANCH:

"KVM_IMAGE_BRANCH: $(BUILD_BRANCH)
MGMT_BRANCH: $(BUILD_BRANCH)"
->
"MGMT_BRANCH: $(BUILD_BRANCH)"